src/main/java/edu/wpi/first/shuffleboard/widget/NetworkTableTreeWidget.java

```diff
@@ -38,7 +39,7 @@
    * Compares tree items alphabetically.
    */
   private static final Comparator<TreeItem<NetworkTableEntry>> alphabetical
-      = Comparator.comparing(item -> item.getValue().getKey().toLowerCase());
+      = Comparator.comparing(item -> item.getValue().getKey().toLowerCase(Locale.getDefault()));
```

Comment:
Yes it is.  See the Gradle release notes:

https://docs.gradle.org/current/release-notes.html#version-of-pmd-has-been-upgraded